src/herder/Herder.h

```diff
@@ -173,7 +173,11 @@ class Herder
     virtual SorobanTransactionQueue& getSorobanTransactionQueue() = 0;
 
     virtual bool sourceAccountPending(AccountID const& accountID) const = 0;
+
+    virtual void overrideMaxDexOpsForTxSetTesting(uint32_t newMax) = 0;
 #endif
+    virtual std::optional<uint32_t> getMaxDexTxOps() const = 0;
```

Comment:
A couple of small things here:
1. Could you please add some comments to these function declarations? I know we're not super consistent about this throughout the codebase, but I'm trying to change that.
2. Can you rename `getMaxDexTxOps` to something to indicate that this is the max DEX tx ops *per tx set*?